ConnorAndersonLarson/PlanetParty
# Feature/sortbox setup

# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?


### Are there any known bugs?


### Summary of the changes
Creates a `SortBox` component with radio buttons and an `updateSort` function to sort the planets by whichever radio button is selected. Also updates file structure to create a `components` directory to hold all components.


### What ticket(s) do the changes resolve?
Resolves #12

Please Link Issues to this PR as necessary.

### Notes for the Reviewer
On my branch, all planet images are of Earth, so it's not easy to tell that the sort has occurred. I have a `console.log` inside the `updateSort` function for that purpose. [ETA: after resolving merge conflicts, the planet names are moving around to show the order (but keep in mind it's using fake data, so there's weird behavior like Mercury showing up with the highest number of moons).]


### How to Test Changes?
Try clicking on the various radio buttons and confirm (probably in the console) that the planets have been sorted into a different order.


### Next Iterations
Still need to style the radio buttons the way we discussed.
